src/includes/getting-started-install/android.mdx

```diff
@@ -16,6 +16,6 @@ android {
 
 // Add Sentry's SDK as a dependency.
 dependencies {
-    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java', '4.0.0') }}'
+    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java', '4.2.0') }}'
```

Comment:
missing https://github.com/getsentry/sentry-docs/edit/master/src/platforms/android/integrations/timber.mdx

Reply:
Fixed. Seems that Android docs don't follow the new structure much.

Reply:
yeah it was before the big restructuring, we should introduce the `guides` for Android too but it's only a single integration for now (Timber) so not worth the effort.